### ISSUE TITLE
[Jobs/ems]Suggestion for new emergency vehicles for EMS #1627

### ISSUE THREAD

**Issue by T4NZIM** [InZidiuZ/legacy-assets-issue-tracker#308]
Assurance:
Have you read through the rules from the README.md file in the root folder of this GitHub repository?
Yes.
Summary:
Write a summary of what your suggestion is about.
Bringing new vehicles for EMS DEPT.
Reason:
Why would your suggestion be a good change to the framework?
It's been almost a year, that I'm doing EMS. Though our devs working really hard to bring the best RP server to us and I also appreciate that. Though it's been a long time, the new emergency vehicles given to EMS Department(though it was).
As I'm doing EMS RP in one of the subdivisions of Legacy RP. As in our subdivision/community(Bangladesh), we have like 12 ranks in our department. And it's being really hard to divide all these few vehicles (not really few but somehow) among these ranks.
My suggested vehicles are:

Charger: https://www.gta5-mods.com/vehicles/dodge-charger-rt-ems-for-fivem-and-sp

EMS TAHOE: https://www.gta5-mods.com/paintjobs/fire-suv-textures-red-lights-for-2015-tahoe

Some other vehicles:
https://www.gta5-mods.com/vehicles/2020-fire-command-tahoe
https://www.gta5-mods.com/paintjobs/fire-suv-textures-red-lights-for-2015-tahoe
https://www.gta5-mods.com/vehicles/els-2015-fire-dept-tahoe-add-on

Hence this is ur server, you have to add those things which will be better for everyone.

So with due respect, I'll request the honorable devs, if they're able to bring some new vehicles in EMS DEPT.

**Comment by ekhowan12** [comment]
I think this should be in EMS.

**Comment by Mugdha57** [comment]
give them a tahoe not charger

**Comment by DC0402** [comment]
❤

**Comment by Pasha3744** [comment]
❤

**Comment by Pasha3744** [comment]
Give Them Charger

**Comment by 8017RAZ** [comment]
OwO

**Comment by 8017RAZ** [comment]
Better give this 
![Uploading 2080bda95cdcbe516ec4aa60156d152c218ad7c6.jpeg…]()

**Comment by XypheRs** [comment]
like diye pashe achi xD

**Comment by joevapes3** [comment]
I agree that some of the EMS vehicles are outdated and the vehicle models while sufficient, they could be better. I will take a look at suitable replacements and change them out once a better alternative is found, and liveries are created for them. Thank you for the suggestion. I'll leave this thread open so that I can update when things get underway.

**Comment by T4NZIM** [comment]
Sure and waiting for your reply :) btw Inzi changed title name *New* to *Mew* XD

**Comment by joevapes3** [comment]
After reviewing the current set of EMS vehicles they seem to be the best in their class and were recently updated along with several new ones added. Thanks for this suggestion and closing this out now.
